GANDLF/utils/gandlf_logger.py

```diff
@@ -0,0 +1,87 @@
+import logging
+import yaml
+from pathlib import Path
+from importlib import resources
+import colorlog
+import tempfile
+from GANDLF.utils import get_unique_timestamp
+
+
+def _flush_to_console():
+    formatter = colorlog.ColoredFormatter(
+        "%(log_color)s%(asctime)s - %(levelname)s - %(message)s",
+        datefmt="%Y-%m-%d %H:%M:%S",
+        log_colors={
+            "DEBUG": "blue",
+            "INFO": "green",
+            "WARNING": "yellow",
+            "ERROR": "red",
+            "CRITICAL": "bold_red",
+        },
+    )
+    console_handler = logging.StreamHandler()
+    console_handler.setFormatter(formatter)
+    logging.root.setLevel(logging.DEBUG)
+    logging.root.addHandler(console_handler)
```

Comment:
If the `try` block is removed from the module below, then this function is no longer needed, right?
```suggestion
```

Reply:
yes if we don't want to flush to the console, the function is no longer needed.